src/overlays/actors/ovl_En_Ik/z_en_ik.c

```diff
@@ -13,44 +13,79 @@
 
 typedef void (*EnIkDrawFunc)(struct EnIk*, GlobalContext*);
 
+typedef enum {
+    /* 0x00 */ ARMOR_UNBROKEN,
+    /* 0x01 */ ARMOR_BROKEN
+} EnIkArmorStatus;
```

Comment:
since `drawArmorFlag`/`armorStatusFlag` are bitfields for some reason,
imo it would make more sense to have a single define `#define ARMOR_BROKEN (1 << 0)`,
and check against it or the literal `0` instead of enum values

Reply:
I did not think of that, I agree and think it makes more sense overall.